src/http/frame/mod.rs

```diff
@@ -0,0 +1,31 @@
+/// A helper macro that unpacks a sequence of 4 bytes found in the buffer with
+/// the given identifier, starting at the given offset, into the given integer
+/// type. Obviously, the integer type should be able to support at least 4
+/// bytes.
+///
+/// # Examples
+///
+/// ```rust
+/// let buf: [u8; 4] = [0, 0, 0, 1];
+/// assert_eq!(1u32, unpack_octets_4!(buf, 0, u32));
+/// ```
+macro_rules! unpack_octets_4 {
+    ($buf:ident, $offset:expr, $tip:ty) => (
+        (($buf[$offset + 0] as $tip) << 24) |
+        (($buf[$offset + 1] as $tip) << 16) |
+        (($buf[$offset + 2] as $tip) <<  8) |
+        (($buf[$offset + 3] as $tip) <<  0)
+    );
+}
+
+pub use self::frames::*;
+//use self::testconfig::*;
+pub use self::dataframe::*;
+pub use self::settingsframe::*;
+pub use self::headersframe::*;
```

Comment:
I'm not such a fan of wildcard imports in general, but especially when it comes to rexports, I'd explicitly state what it is that you intend to {ex,im}port.

In this case, it's only the frame struct that needs rexporting, so just explicitly state that.